design/20190708.certificate-request-crd.md

```diff
@@ -210,6 +408,38 @@ non-networking or other non-transient issue then the `Order` is marked as failed
 further processing will take place by the `CertificateRequest` controller on
 this resource.
 
+#### Issuing Controller
+
+Since external issuers have been built before the addition of Approved and
+Denied conditions, the issuing controller needs to be permissive. An external
+issuer may not honour an Approved condition and will sign and set a
+CertificateRequest as being Ready, before the request has been approved. The
+issuing controller must mark issuance as being successful in this case. In
+practice, this means that the issuing controller is never concerned with
+Approved conditions.
+
+External issuers that do not honour Denied conditions will sign
+CertificateRequests, even if they have a Denied condition set. In this case, the
+issuing controller will successfully complete the issuance of the Certificate.
+
+External Issuers and internal issuer that honour the Denied condition will never
+sign CertificateRequests with the Denied condition set, and thus never set the
+Ready condition. In this case, the issuing controller will consider this
+CertificateRequest as failed, set the Issuing condition to False, and set the
+last failure time. This will cause the issuance to be retried at a later date (1
```

Comment:
I know it has already been mentioned above, but it would be worth adding a note about the requirements for approvers that are honoring the Approval API: they must set `Ready=False` on CertificateRequests when they notice that the CR is either `Approved=True` or `Denied=True`

Suugestion:

```suggestion
Ready condition. In this case, the issuing controller will consider this
CertificateRequest as failed, and will set the condition `Issuing=False`
as well as setting the status field `lastFailureTime`. Note that the issuing
controller is not responsible for setting the `Ready=False` condition on
the CertificateRequests; that's the issuer's responsibility.

The Certificate is then retried later (1
```